net-utils/src/sockets.rs

```diff
@@ -293,11 +293,12 @@ pub fn bind_two_in_range_with_offset_and_config(
     }
 
     for port in range.0..range.1 {
-        if let Ok(first_bind) = bind_to_with_config(ip_addr, port, sock1_config) {
+        let first_bind_result = bind_to_with_config(ip_addr, port, sock1_config);
```

Comment:
Why is this needed? Does not look like we do anything interesting in the else branch...